fem/libceed/ceed.cpp

```diff
@@ -302,22 +678,18 @@ static void InitCeedTensorRestriction(const FiniteElementSpace &fes,
    const Array<int>& dof_map = tfe->GetDofMap();
```

Comment:
Same as above: if the integration rule is not needed here, can it be deleted from the argument list?

Reply:
done.